site/book/04-using-functions/01-declarative-function-execution.md

```diff
@@ -213,6 +213,33 @@ pipeline:
         tier: mysql
 ```
 
+## Specifying function `name`
+
+Functions can optionally be named to e.g. express the combined effect
+of `function` and `functionConfig`.
```

Comment:
Suggest changing this sentence to just

```
Functions can optionally be named using the `pipeline.mutators.name` or `pipeline.validators.name` field to identify a function.  
```

I think `e.g. express the combined effect of function and functionConfig` is not a necessary addition. It might be a good convention to follow, but it is not necessarily the one that we recommend.

Reply:
Agree. I adopted your wording.